src/plot_api/plot_api.js

```diff
@@ -3155,6 +3164,11 @@ exports.purge = function purge(gd) {
     var fullData = gd._fullData || [];
     var calcdata = gd.calcdata || [];
 
+    // remove responsive handler
+    if(gd._responsiveChartHandler) {
+        window.removeEventListener('resize', gd._responsiveChartHandler);
+    }
```

Comment:
We also need to `delete gd._responsiveChartHandler` so it would reattach if we make a new plot in this div later on. The rest of this happens in `Plots.purge` (called a few lines down), and the `delete` and `window.removeEventListener` need to be in the same place (since there are a couple of places we call `Plots.purge` outside `Plotly.purge`, so seems like it would be safer to put both in `Plots.purge`